docs/specifications/tests/Nameserver-TP/nameserver11.md

```diff
@@ -0,0 +1,84 @@
+# NAMESERVER11: Test for undefined EDNS OPTION-CODE
```

Comment:
"Invalid" instead of "undefined"? See discussion below.

Reply:
Rather.. i think it should be unknown as defined in the RFC